superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -154,12 +155,13 @@ function ListViewCard({
   coverLeft,
   coverRight,
   actions,
+  bulkSelectEnabled,
 }: CardProps) {
   return (
     <StyledCard
       cover={
         <Cover>
-          <a href={url}>
+          <a href={bulkSelectEnabled ? undefined : url}>
```

Comment:
can we move this logic up into the `renderCard` functions? We can make url optional and not pass it down when bulkSelect is enabled, that way this component doesn't need to know about bulk select.

Reply:
Alternatively, we may just be able to do a 
```js
e.preventDefault();
e.stopPropagation();
```

in the CardWrapper's onClick handler, when bulkSelect is enabled, to prevent events from triggering on the children.